Makefile

```diff
@@ -112,8 +112,8 @@ start-admission-controller:
 start-resource-manager:
 	@./hack/local-development/start-resource-manager
 
-.PHONY: start-operator $(YQ)
-start-operator:
+.PHONY: start-operator
+start-operator: $(YQ)
```

Comment:
Does this approach work? Because for me `make start-gardenlet` and `make check` are always installing yq, `make test` is always installing promtool, etc.

Reply:
Ah, sorry, my bad, I haven't properly read the PR description.